core/tests/protractor_utils/forms.js

```diff
@@ -51,7 +52,7 @@ var GraphEditor = function(graphInputContainer) {
   var createVertex = async function(xOffset, yOffset) {
     var addNodeButton = graphInputContainer.element(
       by.css('.protractor-test-Add-Node-button'));
-    await addNodeButton.click();
+    await action.click('Test Add Node Button', addNodeButton);
```

Comment:
Here and elsewhere, please remove the `Test ` at the start of the element names

Reply:
done!